app/src/main/java/com/framgia/forder/data/model/Cart.java

```diff
@@ -67,6 +82,30 @@ public void setCartItemList(List<CartItem> cartItemList) {
     }
 
     public String getFormatTotal() {
-        return String.format(FORMAT_PRICE, mTotal) + UNIT_MONEY;
+        return String.valueOf(mTotal) + UNIT_MONEY;
```

Comment:
WARNING: [UselessStringValueOf] No need to call String.valueOf to append to a string.
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#UselessStringValueOf